internal/oc/release_test.go

```diff
@@ -711,8 +890,113 @@ var _ = Describe("GetReleaseBinaryPath", func() {
 	})
 })
 
+var _ = Describe("Get flags from help", func() {
+	var (
+		ctrl         *gomock.Controller
+		mockExecuter *executer.MockExecuter
+	)
+
+	BeforeEach(func() {
+		ctrl = gomock.NewController(GinkgoT())
+		mockExecuter = executer.NewMockExecuter(ctrl)
+	})
+
+	AfterEach(func() {
+		ctrl.Finish()
+	})
+
+	DescribeTable(
+		"With correct help text",
```

Comment:
I think we should have a negative test case here too

Especially:
- What if we supply a help command that returns an error (maybe the executable is not installed for example?)
- What happens if the output is not as expected, do we get the correct results?

Reply:
I added a few tests, including one to verify the behavior when the command doesn't exist and another to verify the behavior when the command returns an exit code other than zero. Let me know what you think.